docs/reference-guides/block-api/block-supports.md

```diff
@@ -424,31 +424,40 @@ supports: {
 }
 ```
 
-## fontSize
+## typography
```

Comment:
I actually think it's better to leave "fontSize" (and description) here but just update the code to include the typography key.

Any thoughts here cc @oandregal @JustinyAhin

Reply:
Hey, I do think we need to do with `typography` what we've done for `color`: have typography as a heading 2 and then the sub-keys as heading 3. Same for border, spacing, layout.

A good first step could be to create a new typography section that contains the existing fontSize and lineHeight. Then, add the undocumented ones. Note, however, that `supports.typography: true` does nothing, the properties need to be enabled individually ([code](https://github.com/WordPress/gutenberg/blob/8d258e505ceb35ac256fcf2c52d973dff286a2cf/packages/block-editor/src/hooks/typography.js#L235)).

Reply:
@mkaz and @aaronrobertshaw as per a conversation we've had about documenting block supports that are still experimental or in the process of becoming matured APIs.

Reply:
Oh, brilliant. Hadn't seen Aaron PR. That's the direction we want :)